src/overlays/actors/ovl_En_Bird/z_en_bird.h

```diff
@@ -12,19 +12,19 @@ typedef struct EnBird {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ SkelAnime skelAnime;
     /* 0x0190 */ EnBirdActionFunc actionFunc;
-    /* 0x0194 */ u32 unk_194;
-    /* 0x0198 */ s32 unk_198;
-    /* 0x019C */ s16 unk_19C;
-    /* 0x019E */ char unk_19E[0x2];
-    /* 0x01A0 */ f32 unk_1A0;
-    /* 0x01A4 */ f32 unk_1A4;
-    /* 0x01A8 */ f32 unk_1A8;
-    /* 0x01AC */ f32 unk_1AC;
-    /* 0x01B0 */ f32 unk_1B0;
-    /* 0x01B4 */ f32 unk_1B4;
-    /* 0x01B8 */ f32 unk_1B8;
-    /* 0x01BC */ f32 unk_1BC;
-    /* 0x01C0 */ s16 unk_1C0;
+    /* 0x0194 */ u32 unk_194; // set to 0 but otherwise unused
+    /* 0x0198 */ s32 timer;
+    /* 0x019C */ s16 aniSpeedMod; // read, but kept at 0. Effects animation speed.
+    /* 0x019E */ char unk_19E[0x2]; // align
```

Comment:
```suggestion
```
Should just be able to delete this with no problem. It should auto align since the next element is an `f32`